frontend/app/routes/protected/person-case/review.tsx

```diff
@@ -381,20 +414,117 @@ export default function Review({ loaderData, actionData, params }: Route.Compone
-                    <span>{inPersonSINCase.birthDetails.provinceName}</span>
-                    <span>{inPersonSINCase.birthDetails.countryName}</span>
+                  <p>
+                    {inPersonSINCase.birthDetails.city}, {inPersonSINCase.birthDetails.provinceName},{' '}
+                    {inPersonSINCase.birthDetails.countryName}
                   </p>
                 </DescriptionListItem>
                 <DescriptionListItem term={t('protected:review.multiple-birth')}>
                   <p>{inPersonSINCase.birthDetails.fromMultipleBirth ? t('protected:review.yes') : t('protected:review.no')}</p>
                 </DescriptionListItem>
               </DescriptionList>
-              <InlineLink file="routes/protected/person-case/birth-details.tsx">
+              <InlineLink file="routes/protected/person-case/birth-details.tsx" search={`tid=${loaderData.tabId}`}>
                 {t('protected:review.edit-birth-details')}
               </InlineLink>
             </section>
+
+            {/* Parent details */}
+
+            <section className="space-y-6">
+              <h2 className="font-lato text-2xl font-bold">{t('protected:parent-details.page-title')}</h2>
+              <DescriptionList className="divide-y border-y">
+                <ul>
+                  {inPersonSINCase.parentDetails?.map((parentDetail, index) => (
+                    <li key={`parent-${index + 1}`} className="divide-y border-y">
+                      <h3 className="font-lato text-2xl font-bold">
+                        {t('protected:parent-details.section-title')}
+                        <span className="ml-[0.5ch]">{index + 1}</span>
+                      </h3>
+                      {parentDetail.unavailable && (
+                        <DescriptionListItem term={t('protected:parent-details.details-unavailable')}>
+                          {t('protected:review.yes')}
+                        </DescriptionListItem>
+                      )}
+                      {!parentDetail.unavailable && (
+                        <DescriptionListItem term={t('protected:parent-details.given-name')}>
+                          {parentDetail.givenName}
+                        </DescriptionListItem>
+                      )}
+                      {!parentDetail.unavailable && (
+                        <DescriptionListItem term={t('protected:parent-details.last-name')}>
+                          {parentDetail.lastName}
+                        </DescriptionListItem>
+                      )}
+                      {!parentDetail.unavailable && (
+                        <DescriptionListItem term={t('protected:review.birth-place')}>
+                          <p>
+                            {parentDetail.birthLocation?.city}, {parentDetail.provinceName}, {parentDetail.countryName}
+                          </p>
+                        </DescriptionListItem>
+                      )}
```

Comment:
I think this is a little easier to read (just check `parentDetail.unavailable` once).

```suggestion
                      {!parentDetail.unavailable && (
                        <>
                          <DescriptionListItem term={t('protected:parent-details.given-name')}>
                            {parentDetail.givenName}
                          </DescriptionListItem>
                          <DescriptionListItem term={t('protected:parent-details.last-name')}>
                            {parentDetail.lastName}
                          </DescriptionListItem>
                          <DescriptionListItem term={t('protected:review.birth-place')}>
                            <p>
                              {parentDetail.birthLocation?.city}, {parentDetail.provinceName}, {parentDetail.countryName}
                            </p>
                          </DescriptionListItem>
                        </>
                      )}
```